rfcs/text/0000-roadmaps+planning.md

```diff
@@ -0,0 +1,421 @@
+
+Overlapping timeboxes denote concurrent work.
+
+Timeboxes with a range of days indicate that the work MAY be completed at any point 
+during the time box, but MUST be completed no later than the final day of the time box.
+
+Day 1 in the Release planning timeline begins in the third Week of the third Milestone of the current Release. For example Q4 planning begins in Q3M3W3 and continues into Q4M1. To reiterate, Planning continues into Milestone 1 of a Release as needed.
+
+* Prerequisite: Product Roadmap
+* Prerequisite: Optional Technical Objectives RFC
+* Day 1: Clarifying the Next Release
+* Day 1-15: Sketching the Release
+* Day 1-20: Creating the Engineering Plan
+
+#### Clarifying the Next Release
+[Clarifying the Next Release]:#clarifying-the-next-release
+
+***Previous Release Milestone 3 Week 3.*** The Roadmap Shepherd schedules a two-hour meeting with DCP PM and the Technical Leads to review any incomplete Objectives in the Release, the next Objectives in the DCP Roadmap, and the Technical Objectives RFC.
+
+Incomplete Objectives are the highest priority for the next Release unless emerging conditions have altered the priorities for the previous Release plan.
+
+The outcome from this meeting is the proposed set of Product and Technical Objectives for the next Release. One (and only one) Product Owner MUST be assigned to each proposed Objective. This Product Owner is responsible for refining and driving the Objective to on-time closure in coordination with Technical Leads and other collaborators.
+
+The dates for the next ZenHub Release and Milestones MUST be assigned or updated in this meeting.
+
+DCP PM MUST assign a rotating Release Shepherd to help the Product Owners and Technical Leads observe the processes and practices for the Release as outlined in this RFC. 
+
+#### Sketching the Release
+
+***Next Release Milestone 1 Week 1.*** For each Objective in the Release, its Product Owner MUST create one or more Epics in the *dcp* repo that document User Stories, Acceptance Criteria, and the Deliverable to Demonstrate.
+
+If an Objective is *well-understood*, then the Product Owner, in consultation with the relevant Technical Leads, MUST assign either a Milestone or a T-Shirt Size label for rough costing and capacity planning to the issue. Pre-defined T-Shirt Sizes are:
+
+| Size | Duration                |
+|:----:|-------------------------|
+|  XXS | 1 week                  |
+|  XS  | 2 weeks                 |
+|   S  | 4 weeks (1 X Milestone) |
+|   M  | 8 weeks (2 X Milestone) |
+|   L  | 12 weeks (1 X Release)  |
+|  XL  | 24 weeks (2 X Release)  |
+|  XXL | 36 weeks (3 X Release)  |
+
+Based on rough costing, Technical Leads and their engineering teams SHOULD provide an initial assessment for "fit" - whether any proposed Epics need to be cut or reduced in scope for the Release.
+
+It is strongly RECOMMENDED that engineering teams buffer 15% capacity for emerging issues during a Release.
```

Comment:
@maniarathi Are you (TLs as a collective) onboarding new users currently? Do you do training, etc. with them? I ask because this ties back to the idea in front of Oversight regarding having a frontline support person for the DCP. If we were to have that person, it makes sense to have that person onboard users as well.

Reply:
There's a larger set beyond onboarding new team members that could be considered - time to support community requests such as RFC(s), public holidays, sick time, meetings, etc. @maniarathi - do you believe that 15% slack time is adequate or perhaps should be noted as the minimum?

Reply:
@CaraLMason As TLs we do train our new hires directly.. it's much easier to have a person that you'll likely be working with show you the ropes instead of putting that pressure on one "DCP trainer." There are a couple new folks coming on board at the Broad I hear and I know Ingest brought on a few new folks recently as well.

Even losing people makes a difference: for example this quarter, Metadata is losing Mallory and Dani and I've had to spend a good chunk of time trying to learn as much as I can before they head off. That knowledge transfer time should be accounted for.

@brianraymor I like the idea of a minimum: 15% slack to account for all the times we as engineers say "Well I thought it was going to be a simple change, but then... " as well as sick time/vacation time/etc. and additional slack depending on other activities that may be happening during the quarter like onboarding.

Reply:
Addressed by https://github.com/HumanCellAtlas/dcp-community/pull/92/commits/2947bb4f9f0bc7d9f478f7ec907c8b4b5a8c1c4f